Sources/JsonModel/SerializationFactory.swift

```diff
@@ -72,7 +132,25 @@ open class SerializationFactory {
             return decodedObject
         }
     }
+    
+    /// If required, allow the factory to set up pointers or transform the decoded objects.
+    open func mapDecodedArray<T>(_ objects : [T]) throws -> [T] { objects }
+    
+    /// If required, allow the factory to set up pointers or transform the decoded objects.
```

Comment:
Is this doc comment intentionally the same as the one just above?

Reply:
Yes, one is for the array decoder and the other is for the object decoder. Both do nothing, but allow for setting up stuff as a part of decoding.

Basically, RSDFactory and the decoding we do allows for hooks into the decoding to load resources and those hooks can apply to either setting up relationships between the elements of an array or transforming the object.